mathesar_ui/src/stores/abstract-types/type-configs/uri.ts

```diff
@@ -0,0 +1,34 @@
+import type {
+  AbstractTypeConfigForm,
+  AbstractTypeConfiguration,
+} from '../types';
+
+const DB_TYPES = {
+  MATHESAR_TYPES__URI: 'MATHESAR_TYPES.URI',
+};
+
+const dbForm: AbstractTypeConfigForm = {
+  variables: {},
+  layout: {
+    orientation: 'vertical',
+    elements: [],
+  },
+};
+
+const uriType: AbstractTypeConfiguration = {
+  icon: '?',
+  defaultDbType: DB_TYPES.MATHESAR_TYPES__URI,
+  cell: {
+    type: 'uri',
+  },
+  getDbConfig: () => ({
+    form: dbForm,
+    determineDbTypeAndOptions: () => ({
+      dbType: DB_TYPES.MATHESAR_TYPES__URI,
+      typeOptions: null,
+    }),
+    constructDbFormValuesFromTypeOptions: () => ({}),
+  }),
```

Comment:
This is optional. Not specifying `getDbConfig` will work as expected.

Reply:
Fixed in dd832d8e7a5bff93f7579328de75bdc6ceada701